### PR TITLE
Workflow Automated Test - LRQA-67761 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 1]
```diff
@@ -30,12 +30,91 @@ definition {
...
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Users",
+			panel = "Control Panel",
+			portlet = "Users and Organizations");
+
+		User.addCP(
+			userEmailAddress = "AUser@liferay.com",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:52]: Could you please change this to the JSON.AddUser macro please?

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 2]
```diff
@@ -30,12 +30,91 @@ definition {
...
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry A",
+			entrySubtitle = "Subtitle",
+			entryTitle = "First Blogs Entry Title");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry B",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:90]: Could you please change this to the JSON.AddBlog macro please?

**Comment by evanilson-santana** [comment]
Fixed.

**Comment by evanilson-santana** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
[Sent to BChan](https://github.com/CalumR23/liferay-portal/pull/19)

**Pull request closed without merging**